common/clock/event_timer_gate_test.go

```diff
@@ -40,7 +40,7 @@ type (
 	}
 )
 
-func TesteventTimerGeteSuite(t *testing.T) {
+func TestEventTimerGeteSuite(t *testing.T) {
```

Comment:
Go 1.24 complains about "Test[lowercase]", but Go 1.23 (in CI) does not.

So this is just a fix for 1.24 users, and it'll be enforced when we update the go.work toolchain / ci / etc.